example/modules/example.ts

```diff
@@ -160,9 +160,9 @@ thin=${res.thin}`
         } catch (error) {
             msg.channel.send(
                 "error! " +
-                    (error || "")
+                    (error as Error || "")
                         .toString()
-                        .split(this.client.token)
+                        .split(this.client.token || "")
```

Comment:
[`split()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split) accepts 0 arguments and will act as a no-op in that case, so this changes no behavior:
> If separator is omitted or does not occur in str, the returned array contains one element consisting of the entire string.

Reply:
this token can be null. so the input is wrong here, that's why added `""` if token is not available

Reply:
you can not build your source code with `tsc`, without this change.

Reply:
It works fine on my machine™ (on the old version of TS)

Reply:
yes, I bumped ts version to latest stable